### PR TITLE
Fix bug navbar

### PR DESCRIPTION
**Pull request by Mouhamed114**

### CODE REVIEW

**Review by MisterPingouin — CHANGES_REQUESTED** [review]
Menu in french and no dead link.

**Review by aldaitzwild — APPROVED** [review]

File: assets/styles/app.scss [hunk 1]
```diff
@@ -15,12 +16,23 @@ $contentFont: "Hind Madurai";
...
 $backgroundColor: rgb(239, 237, 237);
 $optionDefaultColours: white;
-
+$primaryColor : #EAD07F;
 
 body {
    background-color: $backgroundColor;
 }
 
+<<<<<<< HEAD
```
Comment (aldaitzwild) [inline assets/styles/app.scss]: Unresolved conflict

File: assets/styles/app.scss [hunk 2]
```diff
@@ -15,12 +16,23 @@ $contentFont: "Hind Madurai";
...
+.custom-navbar {
+    border-bottom-color: $primaryColor; 
+  
+}
+
+.custum-bg-color-burger-menu{
+    background-color: $primaryColor; 
+  
+}
+=======
```
Comment (aldaitzwild) [inline assets/styles/app.scss]: Unresolved conflict

File: assets/styles/app.scss [hunk 3]
```diff
@@ -230,3 +242,4 @@ body {
 
 
 
+>>>>>>> de1ac5598a869ef8f911279644d8cd4326a2811d
```
Comment (aldaitzwild) [inline assets/styles/app.scss]: Unresolved conflict

File: templates/includes/_navbar.html.twig [hunk 4]
```diff
@@ -30,7 +35,11 @@
     <div class="hidden w-56 absolute top-full right-0 mt-0" id="navbar-hamburger">
       <ul class="flex flex-col font-medium mt-4 rounded-lg bg-gray-50 dark:bg-gray-800 dark:border-gray-700">
         <li>
+<<<<<<< HEAD
```
Comment (aldaitzwild) [inline templates/includes/_navbar.html.twig]: Unresolved conflict

File: templates/includes/_navbar.html.twig [hunk 5]
```diff
@@ -30,7 +35,11 @@
...
+          <a href="#" class="custum-bg-color-burger-menu block py-2 pl-3 pr-4 text-white rounded dark:bg-blue-600" aria-current="page">Galery</a>
+=======
```
Comment (aldaitzwild) [inline templates/includes/_navbar.html.twig]: Unresolved conflict

File: templates/includes/_navbar.html.twig [hunk 6]
```diff
@@ -30,7 +35,11 @@
...
           <a href="{{ path('app_picture_index') }}" class="block py-2 pl-3 pr-4 text-white bg-yellow-200 rounded dark:bg-blue-600" aria-current="page">Galery</a>
+>>>>>>> de1ac5598a869ef8f911279644d8cd4326a2811d
```
Comment (aldaitzwild) [inline templates/includes/_navbar.html.twig]: Unresolved conflict

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: migrations/Version20230607144737.php [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('CREATE TABLE picture (id INT AUTO_INCREMENT NOT NULL, reference VARCHAR(255) DEFAULT NULL, title VARCHAR(255) NOT NULL, subtitle VARCHAR(255) DEFAULT NULL, date VARCHAR(255) DEFAULT NULL, technic VARCHAR(255) DEFAULT NULL, size VARCHAR(255) NOT NULL, category VARCHAR(255) DEFAULT NULL, number INT DEFAULT NULL, comment LONGTEXT DEFAULT NULL, link VARCHAR(300) DEFAULT NULL, image VARCHAR(255) NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
+        $this->addSql('CREATE TABLE messenger_messages (id BIGINT AUTO_INCREMENT NOT NULL, body LONGTEXT NOT NULL, headers LONGTEXT NOT NULL, queue_name VARCHAR(190) NOT NULL, created_at DATETIME NOT NULL, available_at DATETIME NOT NULL, delivered_at DATETIME DEFAULT NULL, INDEX IDX_75EA56E0FB7336F0 (queue_name), INDEX IDX_75EA56E0E3BD61CE (available_at), INDEX IDX_75EA56E016BA31DB (delivered_at), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
```
Comment (aldaitzwild) [inline migrations/Version20230607144737.php]: Why do you have a migration when you don't work on the entities ?

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: assets/styles/app.scss [hunk 8]
```diff
@@ -15,12 +16,21 @@ $contentFont: "Hind Madurai";
...
 body {
    background-color: $backgroundColor;
 }
 
+.custom-navbar {
+    border-bottom-color: $primaryColor; 
+  
+}
+
+.custum-bg-color-burger-menu{
```
Comment (aldaitzwild) [inline assets/styles/app.scss]: I guess you wanted to write "custom" right ?

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: templates/includes/_navbar.html.twig [hunk 9]
```diff
@@ -30,17 +35,10 @@
...
-        </li>
-        <li>
-          <a href="#" class="block py-2 pl-3 pr-4 text-gray-900 rounded hover:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white">Other</a>
-        </li>
-        <li>
-          <a href="#" class="block py-2 pl-3 pr-4 text-gray-900 rounded hover:bg-gray-100 dark:text-gray-400 md:dark:hover:text-white dark:hover:bg-gray-700 dark:hover:text-white">Other</a>
-        </li>
-        <li>
-          <a href="#" class="block py-2 pl-3 pr-4 text-gray-900 rounded hover:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white">Other</a>
+          <a href="{{ path('app_picture_index') }}" class="custum-bg-color-burger-menu block py-2 pl-3 pr-4 text-white rounded dark:bg-blue-600" aria-current="page">Galerie</a>
```
Comment (aldaitzwild) [inline templates/includes/_navbar.html.twig]: Still "custum" ?

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**